File: app/src/main/java/com/magenta/rx/rxa/model/TranslateAnswer.java
package com.magenta.rx.rxa.model;

import java.util.Arrays;

public class TranslateAnswer {

    private String code;
    private String lang;
    private String[] text;

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getLang() {
        return lang;
    }

    public void setLang(String lang) {
        this.lang = lang;
    }

    public String[] getText() {
        return text;
    }

    public void setText(String[] text) {
        this.text = text;
    }

    public String toString() {
        return "{\"code\":" + code + ",\"lang\":" + lang + ",\"text\":[" + Arrays.toString(text) + "]}";
    }
}

File: app/src/main/kotlin/com/magenta/rx/kotlin/record/TranslateAnswer.kt
package com.magenta.rx.kotlin.record

class TranslateAnswer(private var code: String?, private var lang: String?, private var text: Array<String>?) {

    fun getCode() = code

    fun getLang() = lang

    fun getText() = text

    fun setCode(code: String) {
        this.code = code
    }

    fun setLang(lang: String) {
        this.lang = lang
    }

    fun setText(text: Array<String>) {
        this.text = text
    }
}